### PR TITLE
Misc leftover changes discovered while updating protobuff version

### PR DESCRIPTION
**Pull request by lbergelson**
Moving us off of of the weird beta protobuf version to the newest 3.x release.

### CODE REVIEW

**Comment by lbergelson** [comment]
@droazen This is good now I think.

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,7 @@ env:
   - SCALA_VERSION=2.11 TEST_TYPE=wdlGen
   global:
   #gradle needs this
+  - NO_GCE_CHECK=true
```
Comment (lbergelson) [inline .travis.yml:24]: This is irrelevant but we might as well put it in.  Hmn, I should move it above the comment though...

File: src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java [hunk 2]
```diff
@@ -302,7 +302,7 @@ public static void assertGenotypesAreEqual(final Genotype actual, final Genotype
 
     public static void assertGenotypesAreEqual(final Genotype actual, final Genotype expected, final List<String> extendedAttributesToIgnore) {
         Assert.assertEquals(actual.getSampleName(), expected.getSampleName(), "Genotype names");
-        Assert.assertTrue(CollectionUtils.isEqualCollection(actual.getAlleles(), expected.getAlleles()), "Genotype alleles");
+        Assert.assertTrue(CollectionUtils.isEqualCollection(actual.getAlleles(), expected.getAlleles()), "Genotype alleles: " + actual.getAlleles() + " != " + expected.getAlleles());
```
Comment (lbergelson) [inline src/testUtils/java/org/broadinstitute/hellbender/testutils/VariantContextTestUtils.java:305]: This change came up while testing things but I figured it's useful so I'd keep it.

**Comment by droazen** [comment]
Most of these dependency changes are reflected in https://github.com/broadinstitute/gatk/pull/6759. We should let that PR go in first, then rebase this one.

**Comment by droazen** [comment]
@lbergelson Now that https://github.com/broadinstitute/gatk/pull/6759 is merged, can you rebase this branch onto master to reconcile it with the extensive dependency changes in that PR?

**Comment by lbergelson** [comment]
@droazen there are 3 minor changes here that were left over after the other branch got merged.  The only controversial one might be unforcing the protobuff version, but I think it makes sense to just let it float at whatever the other dependencies require since that's what everything else does.

**Comment by lbergelson** [comment]
Closing this.  Moved the 1 relevant change to https://github.com/broadinstitute/gatk/pull/8725

**Pull request closed without merging**